docs/installation/config/env_config.rst

```diff
@@ -0,0 +1,284 @@
+.. _installation_env_config:
+
+===================================
+Environment configuration reference
+===================================
+
+Open Zaak can be ran both as a Docker container or directly on a VPS or
+dedicated server. It relies on other services, such as database and cache
+backends, which can be configured through environment variables.
+
+Available environment variables
+===============================
+
+Required
+--------
+
+* ``DJANGO_SETTINGS_MODULE``: which environment settings to use. Available options:
+  * ``openzaak.conf.production``
+  * ``openzaak.conf.staging``
+  * ``openzaak.conf.docker``
+  * ``openzaak.conf.dev``
+  * ``openzaak.conf.ci``
+
+* ``SECRET_KEY``: secret key that's used for certain cryptographic utilities. You
+  should generate one via
+  [miniwebtool](https://www.miniwebtool.com/django-secret-key-generator/)
```

Comment:
References to external URLs are different in rst, the markdown syntax doesn't work unfortunately:

![image](https://github.com/open-zaak/open-zaak/assets/29249171/707714e6-aa9f-4141-a2f9-ccbb5c3342fe)]

Here is an example how URLs work in rst: https://github.com/open-zaak/open-zaak/blob/main/docs/introduction/architecture.rst?plain=1#L5 (the configured URL is at the bottom of the file)